spec/models/article_spec.rb

```diff
@@ -577,8 +577,9 @@ def build_and_validate_article(*args)
       end
 
       it "keeps published at when trying to set published_at" do
-        new_body_markdown = "---\ntitle: Title\npublished: false\npublished_at:2022-12-05 18:00 +0300---\n\n"
+        new_body_markdown = "---\ntitle: Title\npublished: false\npublished_at: 2022-05-12 18:00 +0300---\n\n"
```

Comment:
Added a space so that the markdown was parsable.

Reply:
Just curious: Looks like you also moved the `published_at` to be in the past, instead of the future? Is that potentially confusing -- like, does it open the possibility of another false positive here?

Reply:
I suppose that initially I meant it to be in the past (but mixed up the order :D)
In this test the date shouldn''t matter, but I have changed so that it would be already in the past to be sure that the test will continue passing after December, 5th